src/All/All.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Description>GraphQL Server meta package with all the packages you need to get started</Description>
+    <PackageTags>GraphQL;transport;server;WebSockets;subscriptions;middleware;authentication;authorization</PackageTags>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Authorization.AspNetCore\Authorization.AspNetCore.csproj" />
+    <ProjectReference Include="..\Core\Core.csproj" />
+    <ProjectReference Include="..\Transports.AspNetCore.NewtonsoftJson\Transports.AspNetCore.NewtonsoftJson.csproj" />
+    <ProjectReference Include="..\Transports.AspNetCore.SystemTextJson\Transports.AspNetCore.SystemTextJson.csproj" />
+    <ProjectReference Include="..\Transports.AspNetCore\Transports.AspNetCore.csproj" />
+    <ProjectReference Include="..\Transports.Subscriptions.Abstractions\Transports.Subscriptions.Abstractions.csproj" />
+    <ProjectReference Include="..\Transports.Subscriptions.WebSockets\Transports.Subscriptions.WebSockets.csproj" />
+    <ProjectReference Include="..\Ui.Altair\Ui.Altair.csproj" />
+    <ProjectReference Include="..\Ui.GraphiQL\Ui.GraphiQL.csproj" />
+    <ProjectReference Include="..\Ui.Playground\Ui.Playground.csproj" />
+    <ProjectReference Include="..\Ui.Voyager\Ui.Voyager.csproj" />
```

Comment:
Regarding https://github.com/graphql-dotnet/server/issues/148#issue-350071286 : I specifically included all UI bits in the metapackage. I see no reason not to include them. These packages take up little space (100KB total) and still provide rich UI.